README.md

```diff
@@ -8,74 +8,116 @@ Head onto our [Discord](https://discord.gg/HM8hhbGSjd) for extended support on b
-
-#### Linux
-	apt-get install libgl1-mesa-dev xorg-dev libglu1-mesa-dev libxinerama-dev libxcursor-dev
-	./bootstrap-vcpkg.sh
-	[OPTIONAL] ./vcpkg integrate install
-	
-#### Install packges
-The VCPKG toolchain file will install the packages found in `vcpkg.json`, see [VCPKG CMake Integration](https://vcpkg.readthedocs.io/en/latest/users/integration/#cmake) and about [Manifest Mode](https://vcpkg.readthedocs.io/en/latest/users/manifests/).
-If manual installation of packages is wanted/desired the following will install all packages.
-	`./vcpkg install asio bullet3 glew glfw3 glm lua openal-soft protobuf zlib spdlog imgui sol2`
-
-### Part 2 (Building)
-1. `git submodule update --init` in the root directory.
-1. `mkdir build/` in to root directory
-1. `cd build/`
-#### Windows (Visual Studio 2019)
-   1. Run cmake-gui setting the source line to the root directory and the build line to the build directory.
-   1. Hit configure and select `Specify toolchain file for cross-compiling` using `**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake`.
-   1. Click generate; then open and build the solution in Visual Studio.
-   1. In the project properties for `trillek-client` change the `Debugging`->`Working Directory` to `$(SolutionDir)..\`.
-   1. **Potentially** Download and install oalinst.zip [OpenAL installer](http://openal.org/downloads/) and install it.
-#### Linux
-##### VCPKG (G++ 7 or CLang 4)
-   1. `cmake -DCMAKE_TOOLCHAIN_FILE=**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake ..` in the build directory
-   1. `make` in the build directory
-##### NON-VCPKG - G++ 7 or CLang 4 (Unsupported due to cmake)
-   1. **INSTALL REQUIRED LIBS** bullet, glew, glfw3, glm, asio, lua, openal-soft, spdlog, Dear ImGui, sol3(also known as sol2), and protobuf. Some of these will need versions not in your distribution (just ask for help in the IRC or Discord.)
-      1. If you are on Ubuntu/Debian/etc. (something with `apt`):
-         1. Run `apt-get install libglew-dev libglfw3 libglm-dev libasio-dev`
-         2. Run `apt-get install liblua5.2-dev libopenal-dev  libbullet-dev`
-         3. Run `apt-get install libprotobuf-dev protobuf-compiler libspdlog-dev`
-      1. If you are on Arch/etc. (something with `pacman`):
-         1. Run `pacman -S glew glfw-x11 glm asio lua52 openal bullet protobuf spdlog`
-   1. `cmake ..` in the build directory
-   1. `make tec` in the build directory
-
-#### Mac OS
-   1. `cmake -DCMAKE_TOOLCHAIN_FILE=**VCPKG_DIR**/scripts/buildsystems/vcpkg.cmake ..` in the build directory
-   1. `make` in the build directory
-
-### Part 3 (Unit Tests)
-To generate the unit tests, follow the same instructions from before, but set to true the flag `BUILD_TESTS_TEC`
-
-## Clang Format
+TEC requires cmake 3.19, a c++17 compatible compiler, and a few libraries GLFW3, GLM, ASIO, Protobuf, GLEW, Lua, Bullet, Dear ImGui, sol3, Spdlog and OpenAL.
+
+The libraries are automatically installed when using CMake and VCPKG manifest mode.
```

Comment:
Manifest mode in VS may require going to Project Properties > vcpkg and enabling it.

Reply:
I cannot find a setting around this, and manifest mode should be solely in the realm of VCPKG https://vcpkg.readthedocs.io/en/latest/users/manifests/